config/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
I'm not sure if this slipped in during the migration by accident. I lost track, are we still using a webhook ? Or do we have removed it because of the new validation rules ?

Reply:
Maybe an issue with rebase, this folder doesn't exist in main https://github.com/llamastack/llama-stack-k8s-operator/tree/main/config